include/mull/Reporters/MutationTestingElementsReporter.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "Reporter.h"
+
+#include <string>
+
+namespace mull {
+
+class SourceInfoProvider;
+class MutationPoint;
+
+class MutationTestingElementsReporter : public Reporter {
+  std::string outputPath;
+  SourceInfoProvider &sourceInfoProvider;
+
+public:
+  MutationTestingElementsReporter(std::string outputPath,
+                                  SourceInfoProvider &sourceInfoProvider);
+  void reportResults(const Result &result, const RawConfig &config,
+                     const Metrics &metrics) override;
+};
```

Comment:
I don't think we agreed on this explicitly, but so far the style is to put public methods first, and then private methods, i.e.:
```
class MutationTestingElementsReporter : public Reporter {
public:
  MutationTestingElementsReporter(...);
  void reportResults(...) override;
private:
  std::string outputPath;
  SourceInfoProvider &sourceInfoProvider;
};
```